python-checks/src/test/resources/checks/duplicateArgument.py

```diff
@@ -82,8 +82,8 @@ def rectangle((top, left), (width, height)): ...
 
 def function_from_typeshed():
```

Comment:
I have the feeling this commit could be left out of this PR and merged separately, as it is unrelated to the ticket.